oleesir/diary-reload
# #161036156 Fix username and email input field

#### What does this PR do?
fix username and email input field

#### Description of Task to be completed?
* fix signin and signup template

#### How should this be manually tested?
* Navigate to the UI folder and open signin.html in your browser
* You will see a sign in template

#### What are the relevant pivotal tracker stories?
[#161036156](https://www.pivotaltracker.com/story/show/161036156)

#### Any background context you want to add?
N/A

#### Screenshots
<img width="1280" alt="screen shot 2018-10-06 at 3 58 04 pm" src="https://user-images.githubusercontent.com/17679511/46572607-e3d23080-c980-11e8-9215-c12bd05c88b9.png">

<img width="1280" alt="screen shot 2018-10-06 at 3 58 09 pm" src="https://user-images.githubusercontent.com/17679511/46572599-d452e780-c980-11e8-95b0-cecb0cd9ff77.png">
